Libraries/Components/TextInput/TextInput.js

```diff
@@ -267,32 +335,42 @@ const TextInput = React.createClass({
      */
     onLayout: PropTypes.func,
     /**
-     * The string that will be rendered before text input has been entered
+     * The string that will be rendered before text input has been entered.
      */
     placeholder: PropTypes.string,
     /**
-     * The text color of the placeholder string
+     * The text color of the placeholder string.
      */
     placeholderTextColor: ColorPropType,
     /**
-     * If true, the text input obscures the text entered so that sensitive text
-     * like passwords stay secure. The default value is false.
+     * If `true`, the text input obscures the text entered so that sensitive text
+     * like passwords stay secure. The default value is `false`.
      */
     secureTextEntry: PropTypes.bool,
     /**
-    * The highlight (and cursor on ios) color of the text input
+    * The highlight (and cursor on iOS) color of the text input.
     */
     selectionColor: ColorPropType,
     /**
-     * See DocumentSelectionState.js, some state that is responsible for
-     * maintaining selection information for a document
+     * An instance of `DocumentSelectionState`, this is some state that is responsible for
+     * maintaining selection information for a document.
+     *
+     * Some functionality that can be performed with this instance is:
+     *
+     * - `blur()`
+     * - `focus()`
+     * - `update()`
+     *
+     * > You can refernce `DocumentSelectionState` in
```

Comment:
Nit: reference 

(saw this in an email, wanted to comment -- hope it's not too late)